Tests/SentryTests/Integrations/Log/FlushLogsIntegrationTests.swift

```diff
@@ -0,0 +1,105 @@
+@_spi(Private) @testable import Sentry
+@_spi(Private) import SentryTestUtils
+import XCTest
+
+#if os(iOS) || os(tvOS) || targetEnvironment(macCatalyst)
+final class FlushLogsIntegrationTests: XCTestCase {
+    
+    private class Fixture {
+        let options: Options
+        let client: TestClient
+        let hub: SentryHubInternal
+        let dependencies: SentryDependencyContainer
+        let notificationCenterWrapper: TestNSNotificationCenterWrapper
+        
+        init() throws {
+            options = Options()
+            options.dsn = TestConstants.dsnForTestCase(type: FlushLogsIntegrationTests.self)
+            options.enableLogs = true
+            
+            client = TestClient(options: options)!
+            hub = TestHub(client: client, andScope: nil)
+            dependencies = SentryDependencyContainer.sharedInstance()
+            notificationCenterWrapper = TestNSNotificationCenterWrapper()
+            dependencies.notificationCenterWrapper = notificationCenterWrapper
+        }
+        
+        func getSut() -> FlushLogsIntegration<SentryDependencyContainer>? {
+            return FlushLogsIntegration(with: options, dependencies: dependencies)
+        }
+    }
+    
+    private var fixture: Fixture!
+    private var sut: FlushLogsIntegration<SentryDependencyContainer>?
+    
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+        fixture = try Fixture()
+        SentrySDKInternal.setCurrentHub(fixture.hub)
+    }
+    
+    override func tearDown() {
+        super.tearDown()
+        clearTestState()
```

Comment:
`m`: Please avoid using the `clearTestState` if you can because it does so many things. If you can, consider only unsetting the things you need to unset. In this case, it should be `SentrySDKInternal.setCurrentHub(nil)`.